test/test_distributions.py

```diff
@@ -250,8 +252,8 @@ def is_all_nan(tensor):
             'high': Variable(torch.ones(1), requires_grad=True),
         },
         {
-            'low': torch.Tensor([1.0, 1.0]),
-            'high': torch.Tensor([2.0, 3.0]),
+            'low': variable([1.0, 1.0]),
+            'high': variable([2.0, 3.0]),
```

Comment:
any reason why these two are `variable` instead of `Variable`?

Reply:
It is a constructor difference, right? I thought it would be unnecessary to wrap `[1.0, 1.0]` with a `Tensor` and then with `Variable`.

However, to maintain consistency, I am changing this. Thanks for pointing this out.